README.md

```diff
@@ -32,22 +40,24 @@ Generated sync functions are compatible with all Sync Gateway 1.x versions.
 
 ### Specifications
 
-Document definitions must conform to the following specification. See the `samples/` directory for some example definitions.
+Document definitions must conform to the following specification. See the `samples/` directory and Kashoo's official [document definitions](https://github.com/Kashoo/kashoo-document-definitions) repository for some examples.
 
 At the top level, the document definitions object contains a property for each document type that is to be supported by the Sync Gateway bucket. For example:
 
-    {
```

Comment:
I removed superfluous indentation from a number of code blocks in the README. You may prefer to view its changes with `?w=1`.